policy/modules/admin/portage.fc

```diff
@@ -8,6 +8,7 @@
 /usr/bin/glsa-check	--	gen_context(system_u:object_r:portage_exec_t,s0)
 /usr/bin/layman	--	gen_context(system_u:object_r:portage_fetch_exec_t,s0)
 /usr/bin/sandbox	--	gen_context(system_u:object_r:portage_exec_t,s0)
+/usr/bin/emerge --	gen_context(system_u:object_r:portage_exec_t,s0)
```

Comment:
hmm this already exists in the gentoo policy. Its actually a bit useless, in gentoo there is also `/usr/lib/python-exec/python[0-9]\.[0-9]*/emerge --      gen_context(system_u:object_r:portage_exec_t,s0)`

But keep this for now then i'll handle the merge conflict when I pull these into the gentoo policy and then i'll upstream the differences after.

Reply:
Haha alright but if you want this kept I'll keep it in. I'm still very new to policy development.